content/enterprise_influxdb/v1.9/administration/cluster-commands.md

```diff
@@ -673,7 +673,7 @@ Successfully left cluster
 ### `remove-data`
 
 Removes a data node from a cluster.
-Use `remove-data` instead of the [`leave`](#leave) argument if you set up your InfluxDB Enterprise cluster with the [Production Installation](/enterprise_influxdb/v1.9/production_installation/) process.
+Use `remove-data` instead of the [`leave`](#leave) argument if you set up your InfluxDB Enterprise cluster with the [installation](/enterprise_influxdb/v1.9/installation/) process.
```

Comment:
Let's look at this content a bit closer...given we're now only documenting one "install process," we may no longer need this caveat? If there's a distinction to be made, we should clarify in what cases you'd use leave vs remove data...